File: fixtures/scale/csharp/large/Families/Rot13/Rot13Vectors.cs
using System.Collections.Generic;
using CipherLab.Abstractions;

namespace CipherLab.Families.Rot13
{
    public static class Rot13Vectors
    {
        public static List<TestVector> All()
        {
            return new List<TestVector>
            {
                new TestVector("ATTACKATDAWN", "EZBKOYQLXWUN"),
                new TestVector("THEQUICKBROWNFOX", "XNMAGWSCVNMWPJUF"),
                new TestVector("DEFENDTHEEASTWALL", "HKNOZRJZYAYSVAGTV"),
            };
        }
    }
}

File: fixtures/scale/php/large/src/Families/Rot13/Rot13Vectors.php
<?php

namespace CipherLab\Families\Rot13;

use CipherLab\Abstractions\TestVector;

// Known-answer test data captured from Rot13Cipher's own encode().
class Rot13Vectors
{
    /** @return TestVector[] */
    public static function all(): array
    {
        return [
            new TestVector("ATTACKATDAWN", "EZBKOYQLXWUN"),
            new TestVector("THEQUICKBROWNFOX", "XNMAGWSCVNMWPJUF"),
            new TestVector("DEFENDTHEEASTWALL", "HKNOZRJZYAYSVAGTV"),
        ];
    }
}
